client/entrypoints/payment-request-settings/payment-request-section.js

```diff
@@ -112,9 +112,12 @@ const buttonThemeOptions = [
 ];
 
 const PaymentRequestsSection = () => {
-	const [ buttonType, setButtonType ] = usePaymentRequestButtonType();
-	const [ size, setSize ] = usePaymentRequestButtonSize();
-	const [ theme, setTheme ] = usePaymentRequestButtonTheme();
+	const [
+		buttonType,
+		handleButtonTypeChange,
+	] = usePaymentRequestButtonType();
+	const [ size, handleSizeChange ] = usePaymentRequestButtonSize();
+	const [ theme, handleThemeChange ] = usePaymentRequestButtonTheme();
```

Comment:
I really don't like this part of the changes. All the `useX` hooks are intended to function as local state — not event handlers. They're setters.

It feels really bad to change the established convention of `const [ x, setX ] = useX();` just to satisfy a linter. Sure; we use some of the setters as event handlers, but that doesn't actually make them event handlers; it's just a shortcut we use to write less code.

What can we do to go back to state hooks using the `[ x, setX ]` pattern instead of `[ x, handleXChange ]`?

Reply:
I'm including in this discussion one solution; wrap the setters:

```jsx
<Elem
    onChange={ ( newX ) => { setX( newX ) }; }
    />
```